crates/environ/src/vmoffsets.rs

```diff
@@ -35,8 +35,8 @@
 // }
```

Comment:
When you have a moment, can you update the above struct pseudo-definition to reflect the changes from this PR? Thanks!

Reply:
I actually added those in a follow up PR [here](https://github.com/bytecodealliance/wasmtime/pull/10265/files#diff-6419ff9508ee5b42d1de13f9509eed5a52ae455d696dedc2b74e090e279757dfR32-R36) when I was touching the VMContext again.